templates/projects/external_config.html

```diff
@@ -3,12 +3,23 @@
 {% set active_link  = "settings" %}
 {% set section = _('External Configurations') %}
 {% from "_formhelpers.html" import render_form %}
+{}
```

Comment:
Empty brackets, was this intentional? :)

Reply:
no, thanks for the catch!